### PR TITLE
Putative fix for broken links on legacy docs pages

### PR DESCRIPTION
**Pull request by dagewa**
Issue as reported by Harry Powell on dials-support.

### CODE REVIEW

**Comment by dagewa** [comment]
Tried to build locally to test, but couldn't actually find the `dials-2.2` pages :thinking:

**Comment by dagewa** [comment]
oh right, `--legacy-version=2.2` or something, but I'm not sure how to get that working

**Comment by Anthchirp** [comment]
We don't build the old pages any more. In that sense the easiest fix is just to fix up the existing page.

File: doc/sphinx/_templates/page.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {%- block body %}
 
 {%- if legacy_version -%}
-  <a class="old-documentation" href="{{ pathto(master_doc) }}/../../{{ pagename }}{{ file_suffix }}">
+  <a class="old-documentation" href="{{ pathto(master_doc) | replace('#', '') }}/../../{{ pagename }}{{ file_suffix }}">
```
Comment (Anthchirp) [inline doc/sphinx/_templates/page.html:7]: probably wouldn't fix the issue.

**Pull request closed without merging**